lib/PEX.ts

```diff
@@ -260,10 +260,10 @@ export class PEX {
    */
   public async verifiablePresentationFromAsync(
     presentationDefinition: IPresentationDefinition,
-    selectedCredentials: OriginalVerifiableCredential[],
-    signingCallBack: (callBackParams: PresentationSignCallBackParams) => Promise<IVerifiablePresentation>,
+    selectedCredentials: (IVerifiableCredential | JwtDecodedVerifiableCredential | CompactJWT)[],
```

Comment:
This is literally what OriginalVerifiableCredential is

Reply:
fixed